ingest-app/docs/api.md

```diff
@@ -1192,6 +1197,64 @@ Ingest permissions for granule subscriptions are granted through the provider vi
 For lack of a better ACL, ingest permissions for collection subscription are granted through the SYSTEM OBJECT TAG_GROUP ACL update permission.
 
 %GENERIC-DOCS%
+
+## <a name="publish-drafts"></a> Publish Drafts 
+### <a name="publish-non-variable-draft"></a> Publish Non Variable Draft 
+#### <a name="publish-non-variable-draft-endpoint"></a> /publish/&lt;non-variable-draft-concept-id&gt;/&lt;native-id&gt;
+
+Non variable drafts can be published, i.e. ingested into CMR as non variable concept, through the non variable draft publishing endpoint.
+
+Example: Publish non variable draft OOD1200000005-PROV1 using native-id orderoption1. Note: if Content-Type is passed in, use it, otherwise use the format stored in the database for OOD1200000005-PROV1.  
```

Comment:
Change text to state:
Example: With the exception of variable drafts, publish a draft record such as an order option with the order option draft concept id of OOD1200000005-PROV1 and a new native-id of orderoption1 for the record to be published. Note: The CMR will use the HTTP header Content-Type if it is passed in to describe the to be published records format. Otherwise the CMR will assume the format of the new record to be published is the same as the draft record.

Reply:
done.